modules/ROOT/pages/bloom-installation/bloom-prerequisites.adoc

```diff
@@ -63,3 +63,21 @@ For more information on supported versions of Neo4j, see link:https://support.ne
 ====
 Keep in mind that if the Bloom client is used with the server component, both client and server should be the same exact version.
 ====
+
+[[bloom-plugin-compatibility]]
+=== Bloom-plugin and Neo4j version compatibility
+
+[cols="<,<", options=header]
+|===
+| Bloom plugin version
+| Supported Neo4j version
+
+| 4.x
+| 4.x
```

Comment:
should we be more specific? 4.4 LTS ?

Reply:
We are not supporting anymore 4.0, 4.1, 4.2, 4.3

Reply:
Updated!